Dockerfile

```diff
@@ -49,29 +50,40 @@ RUN curl -L http://nginx.org/download/nginx-${NGINX_VERSION}.tar.gz > /tmp/nginx
     cd /tmp && \
     tar zxf nginx-${NGINX_VERSION}.tar.gz
 
+
 # Fetch and unarchive ngx_small_light module
 RUN curl -L https://github.com/cubicdaiya/ngx_small_light/archive/v${NGX_SMALL_LIGHT_VERSION}.tar.gz > /tmp/ngx_small_light-${NGX_SMALL_LIGHT_VERSION}.tar.gz && \
     cd /tmp && \
     tar zxf ngx_small_light-${NGX_SMALL_LIGHT_VERSION}.tar.gz && \
     cd /tmp/ngx_small_light-${NGX_SMALL_LIGHT_VERSION} && \
     ./setup
 
+
+# Prepare ngx_mruby
+RUN cd /tmp && \
+    git clone https://github.com/matsumotory/ngx_mruby && \
```

Comment:
As far as possible, we want to keep the version of dependent libraries to make _reproducible_ Dockerfile / Docker image.
Luckily ngx_mruby supports semantic versioning and provide tagged releases at [its GitHub Releases page](https://github.com/matsumotory/ngx_mruby/releases).

I recommend to download and use tagged tarball (e.g. https://github.com/matsumotory/ngx_mruby/archive/v1.18.7.tar.gz) instead of `git clone`.
You can refer the ngx_small_light installation step, which is [described at the previous `RUN`](https://github.com/wantedly/nginx-image-server/blob/fc0079c3a99ff09edd2642c35a28a2435d4af763/Dockerfile#L53).